freex/src/components/AddOffer/AddOffer/OfferForm.jsx

```diff
@@ -0,0 +1,141 @@
+import styles from './AddOffer.module.css';
+import { updateSummary } from '../../../utils/updateSummary';
+import { useState } from 'react';
+import CharacterCounter from '../CharacterCounter/CharacterCounter';
+import Skills from '../Skills/Skills';
+import skillsData from '../Skills/skills.json';
+import PaymentMethod from '../PaymentMethod/PaymentMethod';
+import PremiumOption from '../PremiumOption/PremiumOption';
+import Data from '../PremiumOption/PremiumOptionData.json';
+import Summary from '../Summary/Summary';
+import { useNavigate } from 'react-router-dom';
+
+const OfferForm = ({
+  offer,
+  handleSubmit,
+  disabled,
+  chosenSkills,
+  setChosenSkills,
+  submitText,
+}) => {
+  const [titleLength, setTitleLength] = useState('0');
```

Comment:
To raczej powinna być liczba. Jeżeli CharactersCounter potrzebuje to potem wyświetlać jako `string` to potrzebną operację zmiany titleLength i descriptionLength na string trzeba przeprowadzić w CharactersCounter. Komponent nadrzędny nie powinien być odpowiedzialny za trzymanie tego stanu i jeszcze transformację danej. Inna sprawa, że e.target.value.length i tak zwraca wam liczbę.